config/preset.go

```diff
@@ -29,6 +32,10 @@ func (p Preset) String() string {
 		return "dark"
 	case PresetLight:
 		return "light"
+	case PresetPre0021Dark:
+		return "pre-0.0.21-dark"
+	case PresetPre0021Light:
+		return "pre-0.0.21-light"
```

Comment:
I'm not :100: about the naming here. I would go for something like `legacy-dark`, but I think we should also have a pre-0.3.0 color theme for those that really want it, as I think we should change the default color theme.

Wdyt? @prune998

Reply:
I think this is fine... let's keep the `pre-xxx` as we can support for future changes too. As long as it is documented I'm fine with it.